docs/meetings/2023/TechArea20230905.md

```diff
@@ -50,6 +50,7 @@ OSG 23:
     - May require slight updates to VMU repo regex
       (<https://github.com/opensciencegrid/vm-test-runs/blob/master/bin/vmu.py#L96-L119>).
       Worth just trying it
+-   AI (Mat): Examine OSG-Test to see what needs to be updated for OSG 23
```

Comment:
Can you create a ticket for this?

Reply:
https://opensciencegrid.atlassian.net/browse/SOFTWARE-5679